node/node.go

```diff
@@ -62,12 +65,16 @@ type Node struct {
 // NewNode creates a new Node with the provided config.
 func NewNode(config *Config, pubIPProvider pubip.Provider, logger common.Logger) (*Node, error) {
 	// Setup metrics
-	sdkClients, err := buildSdkClients(config, logger)
-	if err != nil {
-		return nil, err
-	}
-	metrics := NewMetrics(sdkClients.Metrics, sdkClients.PrometheusRegistry, logger, ":"+config.MetricsPort)
-	rpcCallsCollector := rpccalls.NewCollector(AppName, sdkClients.PrometheusRegistry)
+	// sdkClients, err := buildSdkClients(config, logger)
```

Comment:
why was this commented out?

Reply:
At the time, the SDK hadn't been updated to reflect the new contracts. Since the client from the SDK was only being used for metrics, I created the metrics separately. 

I'm not sure if it makes sense to build all of the SDK clients if we aren't using them. 

My preference here would be to remove this commented code and create a ticket to track further integration with the SDK.